src/net/sourceforge/kolmafia/request/GenieRequest.java

```diff
@@ -112,11 +115,11 @@ public void run() {
     if (InventoryManager.hasItem(ItemPool.GENIE_BOTTLE)
         && Preferences.getInteger("_genieWishesUsed") < 3) {
       itemId = ItemPool.GENIE_BOTTLE;
-    } else if (InventoryManager.hasItem(ItemPool.POCKET_WISH)) {
-      itemId = ItemPool.POCKET_WISH;
     } else if (KoLCharacter.inLegacyOfLoathing()
```

Comment:
General observation: all over the place, we check for "in LoL and have replica item". Since freeing the king takes you out of LoL and immediately destroys all replica items and familiars, why bother checking the path? Surely possession of the replica item suffices?

Or do we not yet remove all replicas upon freeing the king?

Since you just moved the pocket wish, rather than adding replica checking here, I'm not requesting a change in this PR. We should fix everything when we add code to remove all replicas.

Reply:
It was for performance: as you can only have the replica item in LoL, we don't need to check the entire inventory for the presence of an item unless you're in LoL.